src/libostree/ostree-repo-commit.c

```diff
@@ -2593,16 +2593,6 @@ write_directory_to_mtree_internal (OstreeRepo                  *self,
 
       ostree_mutable_tree_set_metadata_checksum (mtree, ostree_repo_file_tree_get_metadata_checksum (repo_dir));
```

Comment:
I'm unsure whether this `set_metadata_checksum` should also be removed.  I'm a little confused what the metadata checksum is as I don't think there's a parallel in the git world.